public/app/core/components/QueryOperationRow/QueryOperationRowHeader.tsx

```diff
@@ -37,11 +37,10 @@ export const QueryOperationRowHeader = ({
       <div className={styles.column}>
         <IconButton
           name={isContentVisible ? 'angle-down' : 'angle-right'}
-          title="toggle collapse and expand"
+          tooltip={isContentVisible ? 'Collapse query row' : 'Expand query row'}
           aria-label="toggle collapse and expand query row"
           className={styles.collapseIcon}
           onClick={onRowToggle}
-          type="button"
```

Comment:
same here about `type="button"`... either we should keep this, or add it to the base `IconButton`